js/pivx_shielding.js

```diff
@@ -137,21 +144,25 @@ export default class PIVXShielding {
   }
 
   /**
-   * Creates a transaction, sending `amount` satoshis to the address
-   * @param {{address: String, amount: Number}} target
+   * Createes a transaction, sending `amount` satoshis to the address
+   * @param {{address: String, amount: Number, blockHeight: Number, useShieldInputs: bool}} target
```

Comment:
are you sure that blockHeight is needed?

Reply:
Builder takes it as a parameter

Reply:
right but I'm sure that we can just pass a symbolic height in which sapling was activated

Reply:
I think zcash uses the expiry height in their transaction, but PIVX doesn't, so it should in theory work